plugins/modules/aci_epg_to_domain.py

```diff
@@ -75,6 +75,20 @@
     - Name of the VMM Domain Enhanced Lag Policy.
     type: str
     aliases: [ lag_policy ]
+  vmm_uplink_active:
+    description:
+    - A list of active uplink IDs.
+    - The order decides the order in which active uplinks take over for a failed uplink.
+    - At least one active uplink must remain specified in list when a active uplink is previously configured.
```

Comment:
in the list when an active uplink was previously configured.

Reply:
ok